src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+export const OptionsBitmap = {
+    /** Command execution control - see 1.6.5.8.1. */
```

Comment:
nit: either use the full link to the specs or don't reference it.
"see 1.6.5.8.1" is not explicit and will be confusing if displayed by itself (by the IDE).
The spec section is already referenced above, so I don't think there is a need to mention it again here.

Same below